rom/laghos.cpp

```diff
@@ -700,7 +700,7 @@ int main(int argc, char *argv[])
     romOptions.indicatorType = getlocalROMIndicator(indicatorType);
     if (rom_online)
     {
-        std::string filename = testing_parameter_outputPath + "/ROMsol/romS_1";
+        std::string filename = testing_parameter_outputPath + "/ROMsol/romS_1"; // romS_0 is allowed
```

Comment:
This comment is not very clear. A comment about why `romS_1` is checked instead of `romS_0` might be more informative.